data/json/professions.json

```
@@ -4004,5 +4004,19 @@
         ]
       }
     }
+  },
+  {
+    "type": "profession",
+    "ident": "frat",
+    "name": { "male": "Frat Boy", "female": "Frat Girl" },
```

```suggestion
    "name": { "male": "Frat Boy", "female": "Sorority Girl" },
```
I think Fraternities are male only, but I´m not sure.

---

I think that's right. I don't know what would be a better way to pass the meaning I have `Son/daugther of rich parents who does nothing much but spend parents allowance`.